config/initializers/permissions.rb

```diff
@@ -71,6 +71,12 @@
                      require: :loggedin,
                      contract_actions: { users: %i[read update] }
 
+      map.permission :view_user_hover_card,
+                     {
+                       "users/hover_card": %i[show]
+                     },
+                     permissible_on: :global
+
```

Comment:
Really not sure about this one. We *need* to provide some sort of permission when we want to render a component for the hover cards. But which one should it be?

Reply:
Removed in favor of `no_authorization_required!`